scripts/analyze.sh

```diff
@@ -104,6 +104,11 @@ if [ $? -eq 0 ] && [[ $MODULE != *"BUILD_MODULE_BUFFER"* ]]; then
     >&2 echo Using module: Buffer
     MODULES+=" -DBUILD_MODULE_BUFFER"
 fi
+sensor=$(grep -E AccelerometerSensor\|Gyroscope  $SCRIPT)
```

Comment:
Ok, I'll update accordingly, I was wondering that too since it's odd that the one has, but one does not.